src/test/crimson/test_messenger.cc

```diff
@@ -3715,6 +3715,9 @@ int main(int argc, char** argv)
       return test_v2_protocol(v2_test_addr, v2_testpeer_addr, v2_testpeer_islocal);
     }).then([] {
       std::cout << "All tests succeeded" << std::endl;
+      // Seastar has bugs to have events undispatched during shutdown,
```

Comment:
are you sure it's a bug on seastar side instead of on ours ? for instance, we might forget to drain all the tasks in queue before shutting down the reactor?

Reply:
I think so, the bug was related to https://github.com/ceph/seastar/blob/8188d3b8b7091db9227824791d022d64c98193e9/net/posix-stack.hh#L81-L83 , where the LeakSanitizer shows that the task to delete `foreign_ptr<lw_shared_ptr<load_balancer>> _lb` was not scheduled when the program exit very fast.